internal/controller/management/projects/projects.go

```diff
@@ -341,6 +364,194 @@ func (r *reconciler) ensureProjectAdminPermissions(
 	return nil
 }
 
+func (r *reconciler) ensureDefaultProjectRoles(
+	ctx context.Context,
+	project *kargoapi.Project,
+) error {
+	logger := logging.LoggerFromContext(ctx).WithFields(log.Fields{
+		"project":   project.Name,
+		"name":      project.Name,
+		"namespace": project.Name,
+	})
+
+	const adminRoleName = "admin"
+	const viewerRoleName = "viewer"
+	allRoles := []string{adminRoleName, viewerRoleName}
+
+	for _, saName := range allRoles {
+		saLogger := logger.WithField("serviceAccount", saName)
+		if err := r.createServiceAccountFn(
+			ctx,
+			&corev1.ServiceAccount{
+				ObjectMeta: metav1.ObjectMeta{
+					Name:      saName,
+					Namespace: project.Name,
+					Annotations: map[string]string{
+						rbacapi.AnnotationKeyManaged: rbacapi.AnnotationValueTrue,
+					},
+				},
+			},
+		); err != nil {
+			if kubeerr.IsAlreadyExists(err) {
+				saLogger.Debug("ServiceAccount already exists in project namespace")
+				continue
+			}
+			return fmt.Errorf(
+				"error creating ServiceAccount %q in project namespace %q: %w",
+				saName,
+				project.Name,
+				err,
+			)
+		}
+	}
+
+	roles := []*rbacv1.Role{
+		{
+			ObjectMeta: metav1.ObjectMeta{
+				Name:      "admin",
```

Comment:
```suggestion
				Name:      adminRoleName,
```

Reply:
Ah... meant to do this and missed it. Thanks.